css/design_system/utils/colors.scss

```diff
@@ -1,54 +1,60 @@
 $colors: 
-  "--s-color-black", 
-  "--s-color-white", 
-  "--s-color-cyan-400",
-  "--s-color-cyan-500",
-  "--s-color-cyan-600",
-  "--s-color-cyan-700",
-  "--s-color-gray-100",
-  "--s-color-gray-200",
-  "--s-color-gray-300",
-  "--s-color-gray-400",
-  "--s-color-gray-500",
-  "--s-color-gray-600",
-  "--s-color-gray-700",
-  "--s-color-gray-800",
-  "--s-color-red-500",
-  "--s-color-red-600",
-  "--s-color-red-700",
-  "--s-color-yellow-500",
-  "--s-color-yellow-600",
-  "--s-color-yellow-700";
+  "--s-color-black" false, 
```

Comment:
Might be helpful to include a quick explanatory comment above: "List of colors with boolean indicating whether to use a dark background"